src/compass/utils/h5_helpers.py

```diff
@@ -567,13 +567,32 @@ def metadata_to_h5group(parent_group, burst, cfg):
         add_dataset_and_attrs(burst_meta_group, meta_item)
 
     # Add parameters group in processing information
+    dry_tropo_corr = True if (cfg.weather_model_file is not None) and \
+                             ('dry' in cfg.tropo_params.delay_type) else False
+    wet_tropo_corr = True if (cfg.weather_model_file is not None) and \
+                             ('wet' in cfg.tropo_params.delay_type) else False
+    tec_corr = True if cfg.tec_file is not None else False
     par_meta_items = [
-        Meta('ellipsoidalFlatteningApplied', cfg.geocoding_params.flatten,
-             "If True, CSLC-S1 phase has been flatten with respect to a zero height ellipsoid",
-             {'units':'unitless'}),
-        Meta('topographicFlatteningApplied', cfg.geocoding_params.flatten,
-             "If True, CSLC-S1 phase has been flatten with respect to topographic height using a DEM",
-             {'units': 'unitless'}),
+        Meta('ellipsoidal_flattening_applied', bool(cfg.geocoding_params.flatten),
+             "If True, CSLC-S1 phase has been flatten with respect to a zero height ellipsoid"),
+        Meta('topographic_flattening_applied', bool(cfg.geocoding_params.flatten),
+             "If True, CSLC-S1 phase has been flatten with respect to topographic height using a DEM"),
+        Meta('bistatic_delay_applied', bool(cfg.lut_params.enabled),
+             "If True, bistatic delay correction has been applied"),
+        Meta('azimuth_fm_rate_applied', bool(cfg.lut_params.enabled),
+             "If True, azimuth FM-rate mismatch correction has been applied"),
+        Meta('geometry_doppler_applied', bool(cfg.lut_params.enabled),
+             "If True, geometry steering doppler correction has been applied"),
+        Meta('los_solid_earth_tides_applied', bool(cfg.lut_params.enabled),
+             "If True, solid Earth tides correction has been applied in slant range direction"),
```

Comment:
Just wondering if we will need to put a placeholder for sth. like `azimuth_solid_earth_tides_applied`

Reply:
It is not a bad idea. Just in case it will be more difficult to change the product